receiver/pluginreceiver/README.md

```diff
@@ -75,6 +75,8 @@ The following keys are used when defining a parameter.
 
 **Warning**: Parameters must adhere to their definition. Invalid parameters will return an error during configuration.
 
+**Note**: The value given for the timezone parameter type must be a standard UTC timezone, shown in this [timezone list](https://github.com/observIQ/observiq-otel-collector/blob/main/receiver/pluginreceiver/timezone.md).
```

Comment:
This link I think can just be

```markdown
[timezone list](./timzone.md)
```

Right now it includes the branch name so it won't render the correct version on the current branch.